lms/djangoapps/teams/views.py

```diff
@@ -59,6 +62,25 @@
 MAXIMUM_SEARCH_SIZE = 100000
 
 
+@receiver(pre_save, sender=CourseTeam)
+def team_pre_save_callback(sender, instance, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Although to be fair, I guess we don't have any real guarantees that some other signal handler doesn't throw an exception in a post-save callback, thus preventing an event from getting emitted after a successful save. Maybe we're in the same situation either way.